security/seccomp/seccomp.go

```diff
@@ -0,0 +1,141 @@
+// +build linux,cgo,seccomp
```

Comment:
Can we call this file `seccomp_linux.go`? And `unsupported.go` as `seccomp_unsupported.go`?